.release/ami_install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+
+DIRECTORY=`dirname $0`
+source $DIRECTORY/bootstrap_steps.sh
+
+echo "LOADED UP BOOTSTRAP STEPS"
+
+install_wallaroo() {
+  PYTHON_PATH=$(python -c "import site; print(site.getsitepackages()[0])")
+  git checkout "$WALLAROO_VERSION"
+  sudo cp machida/wallaroo.py "$PYTHON_PATH"
+  (make build-machida target_cpu=sandybridge && \
+       sudo sudo cp machida/build/machida /usr/bin)
+  (cd utils/data_receiver && make && sudo cp ./data_receiver /usr/bin)
+  (cd giles/sender && make && sudo cp ./sender /usr/bin/giles_sender)
```

Comment:
This one also needs `target_cpu=sandybridge` added to the `make` command for the same reasons as for the `build-machida` target.